clients/admin-ui/src/features/datastore-connections/system_portal_config/forms/ConnectorParametersForm.tsx

```diff
@@ -238,23 +235,10 @@ const ConnectorParametersForm: React.FC<ConnectorParametersFormProps> = ({
   };
 
   const handleTestConnectionClick = async () => {
-    try {
-      await trigger(connectionConfig!.key).unwrap();
-    } catch (error) {
-      handleError(error);
-    }
+    const result = await trigger(connectionConfig!.key);
+    onTestConnectionClick(result);
```

Comment:
right now `onTestConnectionClick` is typed as `(value: any) => void` which isn't great. I know that already existed before this PR, but maybe while we are here, we can clean that up a bit 🧹 

unfortunately that type is difficult to track down. we can make our own type though which captures the fields we need (in this case `data` and `fulfilledTimeStamp`)

```typescript
export interface TestConnectionResponse {
  data?: DatastoreConnectionStatus;
  fulfilledTimeStamp?: number;
}
```
so then we can update to `onTestConnectionClick: (value: TestConnectionResponse) => void;`

and also in `ConnectorParameters.tsx`, `response` no longer has to be of type `any`, we can import our new `TestConnectionResponse` type instead. it does yield a few typing errors though, but that's a good thing! since we should handle the case where `fulfilledTimeStamp` is undefined, etc.